backend/routers/api/collections.ts

```diff
@@ -0,0 +1,53 @@
+import express from "express";
+import {
+  GetCOLLECTION_DATA,
+  SetCOLLECTION_DATA,
+} from "../../constants/Collections";
+
+const router = express.Router();
+
+// POST /collections
+// Update all collections data
+router.post("/", (req, res) => {
+  SetCOLLECTION_DATA(req.body);
+
+  res.status(200).json(GetCOLLECTION_DATA());
+});
+
+// GET /collections
+// Get all collections data
+router.get("/", (req, res) => {
+  res.status(200).json(GetCOLLECTION_DATA());
+});
+
+// GET /collections/collectionId
+// Get a collection data
+router.get("/:collectionId", (req, res) => {
+  const reqData = {
+    current: req.query.current,
+    size: req.query.size,
+    collectionId: req.params.collectionId,
+  };
+
+  const CollectionData = GetCOLLECTION_DATA().filter(
+    (collection) => collection.urlTitle === reqData.collectionId
+  )[0];
+
+  const resData = {
+    total: Math.ceil(CollectionData.products.length / Number(reqData.size)),
+    collection: {
+      image: CollectionData.image,
+      title: CollectionData.title,
+      urlTitle: CollectionData.urlTitle,
+      content: CollectionData.content,
+      products: CollectionData.products.slice(
+        (Number(reqData.current) - 1) * Number(reqData.size),
+        Number(reqData.current) * Number(reqData.size)
+      ),
+    },
+  };
```

Comment:
```suggestion
  const resData = {
    total: Math.ceil(CollectionData.products.length / Number(reqData.size)),
    collection: {
      image: CollectionData.image,
      title: CollectionData.title,
      urlTitle: CollectionData.urlTitle,
      content: CollectionData.content,
      products: CollectionData.products.slice(
        (Number(reqData.current) - 1) * Number(reqData.size),
        Number(reqData.current) * Number(reqData.size)
      ),
    },
  };
```

This code can be simplified.